### DISCUSSION TITLE
confusion around app and pages folder with global styles

### DISCUSSION THREAD

**Discussion by hp8wvvvgnj6asjm7** [vercel/next.js#52166] [Help]
### Summary

The docs say that we should slowly migrate to app folder
but for using global styles with pages it recommends using _app.tsx

so which one is right now?

**Comment by icyJoseph** [comment]
Both, these two are two separate React trees.

As long as you want to maintain pages, you probably want to keep your `_app` and progressively remove code that supports other pages. The last step of migration should be, the deletion of `_app`.

You can still have a, say, `global.css` file somewhere in your code structure, and import it to the `_app` and `app/layout` files.
